### PR TITLE
Improve the build

### PR DESCRIPTION
**Pull request by j-clark**

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 set -eo pipefail
 
-VERSION=$(npm view athenaeum version)
+BUMP_SIZE=$(buildkite-agent meta-data get bump-size)
+
+git config --global user.name "Randy the PolicyGenius Robot"
+git config --global user.email "robot@policygenius.com"
+git config --global push.default simple
+
+VERSION=$(npm version $BUMP_SIZE)
```
Comment (drewdrewthis) [inline bin/deploy]: Nice. I didn't know this was a thing.

**Comment by j-clark** [comment]
@policygenius/developers This should be ready to go! Stuff in this PR:

- i added the jest tests and the linters
- you can deploy a custom branch to an S3 bucket that you'll be able to view at http://rcl.policygenius.com/yourbranchname-randombuildid/index.html (the exact URL is printed to the console
- on master, you can do a major/minor/patch bump and publish that to npm, as well as deploy the styleguide to http://rcl.policygenius.com/newversion/index.html
- i'm working on adding a script to make http://rcl.policygenius.com browsable so we don't have to have flaky redirects like with the previous styleguide

File: package.json [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
   "scripts": {
     "precommit": "yarn run lint",
     "prepush": "yarn run lint",
     "open": "open http://localhost:6060",
     "c": "yarn run plop create-component",
-    "build": "yarn run lint && ./node_modules/webpack/bin/webpack.js --config ./webpack.config.dev.js --progress --colors",
-    "build:prod": "yarn run lint && ./node_modules/webpack/bin/webpack.js --config ./webpack.config.prod.js --progress --colors",
     "build:publish": "./node_modules/webpack/bin/webpack.js --config ./webpack.config.publish.js --progress --colors",
-    "publish": "yarn run build:publish && yarn run publish",
+    "publish": "yarn run build:publish && npm publish",
```
Comment (drewdrewthis) [inline package.json]: Does `yarn run publish` not work?

Comment (j-clark) [inline package.json]: yes, this is the definition of `yarn run publish`. it was calling itself

Comment (drewdrewthis) [inline package.json]: @j-clark Yeah, I saw that. LGTM

**Pull request merged**